analysis/11.analyze.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    GENE_COL = "gene_symbol_0"
</pre><p>We want to load this from config. <code>config["aggregate"]["perturbation_name_col"]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/brieflow-analysis/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>